<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="push01">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="push24">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="push12">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="push23">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="push04">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="push19">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="push10">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="push16">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="push05">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="push20">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="push03">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="push06">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="push21">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="push22">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="push18">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="push09">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="push13">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="push07">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="push02">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="push11">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="push14">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="push15">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="push08">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="push17">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">butonGrup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="pairsNo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="PairsLabel">
       <property name="font">
        <font>
         <family>American Typewriter</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Pairs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="TriesLabel">
       <property name="font">
        <font>
         <family>American Typewriter</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tries</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="tryNo"/>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="resetButton">
       <property name="font">
        <font>
         <family>American Typewriter</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="butonGrup"/>
 </buttongroups>
</ui>
